tests/fixtures/ship/validate_ship_weirdpackagenames_supported.stderr

```diff
@@ -17,5 +17,4 @@ Caused by:
 
 Caused by:
 ...
-  invalid character `-` in package name: `--pkg`, the first character must be a Unicode XID start character (most letters or `_`)
```

Comment:
Is there a reason to remove this? It seems accepted only to end up in the `...`, but I would prefer if wildcards were used the least often possible.

Reply:
did it because the test failed for me with a slightly differently formatted error.

doesn't occur anymore so reverted the change